data/types/device/alarm.go

```diff
@@ -15,10 +15,15 @@ func init() {
 
 type Alarm struct {
 	AlarmType *string `json:"alarmType" bson:"alarmType" valid:"devicealarmtype"`
+	Status    *string `json:"status,omitempty" bson:"status,omitempty" valid:"-"`
```

Comment:
Do we want to standardize on where the `omitempty` occurs in the tag?  With a quick find I see that the vast majority have it second except for a few validators where it is first (such as `valid:"omitempty,required"`).